src/screens/Login/styles.ts

```diff
@@ -0,0 +1,49 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor } from 'src/common/enums';
+
+const styles = StyleSheet.create({
+  container: {
+    marginHorizontal: 27,
+    marginVertical: 22,
```

Comment:
What are those magic numbers? We might already have default screen padding shared constant somewhere in code. Ask @katyakostikova for more details on this

Reply:
Yes, I've added this in src/styles but it's still in PR #27